cardinal/server/handler/events.go

```diff
@@ -0,0 +1,11 @@
+package handler
+
+import (
+	"github.com/gofiber/contrib/websocket"
+	"github.com/gofiber/fiber/v2"
+	"pkg.world.dev/world-engine/cardinal/events"
+)
+
+func WebSocketEvents(hub events.EventHub) (func(*fiber.Ctx) error, func(*fiber.Ctx) error) {
```

Comment:
This is a simple function, it can be read, and understood after some investigation, but I still think we should rename it to something better. Looks like it returns an upgrader/handler combo?

Reply:
I would personally name it `NewWebSocketUpgraderAndHandlerFunctions` but I named it like the above to cater to the convention here.

Reply:
This function isn't explicitly required either I can just pass the functions from the library directly. It's done in attempt to mimic the existing patterns.

Reply:
Read the code a bit more, I see what you're doing, seems like there are a couple other options but this is no biggie, I'll resolve the comment